src/components/dashboard/ActivityRewardsSection/index.tsx

```diff
@@ -0,0 +1,133 @@
+import type { ReactNode } from 'react'
+import { Typography, Card, IconButton, Link, SvgIcon, Grid, Button } from '@mui/material'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import css from './styles.module.css'
+import { useRemoteSafeApps } from '@/hooks/safe-apps/useRemoteSafeApps'
+import { SafeAppsTag } from '@/config/constants'
+import SafeMilesLogo from '@/public/images/common/safe-miles-logo.svg'
+import classNames from 'classnames'
+import useChainId from '@/hooks/useChainId'
+import { useShareSafeAppUrl } from '@/components/safe-apps/hooks/useShareSafeAppUrl'
+import { useDarkMode } from '@/hooks/useDarkMode'
+
+// Chains
+export const Chains = {
+  MAINNET: '1',
+  SEPOLIA: '11155111',
+}
+
+// Strictly type configuration for each chain above
+type ChainConfig<T> = Record<(typeof Chains)[keyof typeof Chains], T>
+
+const CHAIN_START_TIMESTAMPS: ChainConfig<number> = {
+  1: 1713866400000, // 23rd April 2024
+  11155111: 1709290800000, // 01st March 2024
+}
+
+const JUNE_10_TIMESTAMP = 1718013600000
+const SEPTEMBER_10_TIMESTAMP = 1725962400000
```

Comment:
We need to update the timestamp to September 30. There were also some other minor refactors in the governance app for the dates. I suggest we copy them over when its time to merge this PR.